utils/index.js

```diff
@@ -262,7 +291,9 @@ const sortByLineNumber = (...objToSort) => {
  * @param {Array} routes array of route objects { name: "start", path: "/start" },
  * @returns { path: "" }
  */
-const getPreviousRoute = (path, session, routes = defaultRoutes) => {
+const getPreviousRoute = (req, routes = defaultRoutes) => {
+  const path = req.path
```

Comment:
If you wanted to be really hip, you could be like
```js
const { path, session } = req
```

Reply:
oh, duh 🤦‍♀